cron.php

```diff
@@ -88,6 +88,12 @@
 loadDatabase();
 reloadSettings();
 
+// We need to init some super-default things because there's a lot of code that might accidentally rely on it.
+$user_info = [
```

Comment:
Is that the best way to handle it? It's simpler, that's for sure, but...

Reply:
The problem is, I have no idea where the problem is.

Somewhere in the hulk of code, something is calling timeformat in a cron run. Except I can't find anything that triggers timeformat from any of the cron tasks...

Reply:
Ah, I see what you mean. Cool.